webgui-new/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "webgui-new",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "ts-node src/server.ts",
+    "build": "next build && next export",
+    "start": "NODE_ENV=production ts-node src/server.ts",
+    "lint": "next lint",
+    "thrift-ts": "thrift-typescript --fallbackNamespace none"
+  },
+  "devDependencies": {
+    "@creditkarma/thrift-typescript": "^3.7.6",
```

Comment:
This is a note to myself, to check whether the already used (and *official*) Thrift library why cannot generate the expected TypeScript files.

This should not affect your thesis submission, though, submitting with this solution is also fine.

Reply:
Created #606 to address this later.